addons/captives/CfgVehicles.hpp

```diff
@@ -51,6 +51,7 @@ class CfgVehicles {
                     showDisabled = 0;
                     icon = QPATHTOF(UI\captive_ca.paa);
                     priority = 2.2;
+                    insertChildren = QUOTE([ARR_2(_player, _target)] call DFUNC(addLoadCaptiveActions));
```

Comment:
Just `_this call...` works here as well, so you don't need to do macro shenanigans.

Reply:
`_this` is `[_target, _player]` and @654wak654 wants reversed array (I think).
Also space in `ARR_2` macro. Also see @commy2 comment https://github.com/acemod/ACE3/pull/5384#pullrequestreview-62922317

Reply:
He is calling his own function, may as well revers the order there.

Single quotes are fine but you lose syntax highlighting because it's all a long string then, personal preference. I prefer single quotes if it really makes the thing shorter, but in this case you don't need anything but QUOTE and DFUNC.